Misc/NEWS.d/next/Tests/2019-02-26-12-51-35.bpo-36123.QRhhRS.rst

```diff
@@ -0,0 +1,2 @@
+timeout is now set once the socket is created in :_testWithTimeoutTriggeredSend(self) rather than on the

```

Comment:
In your NEWS entry, it's unclear why you need this change. I prefer to explicit say that you fix a race condition, and mention the test name from the start. Moreover, "_testWithTimeoutTriggeredSend" is the private method, whereas testWithTimeoutTriggeredSend() is the public test name.

Fix test_socket.testWithTimeoutTriggeredSend() race condition: timeout is now set once the socket is created in rather than on the connection.